CHANGELOG.md

```diff
@@ -1,2 +1,9 @@
+- Adds support for the Firestore Provisioning API commands: (#5616)
+- firestore:databases:list
+- firestore:databases:create
+- firestore:databases:get
+- firestore:databases:delete
+- firestore:locations
+- Releases Cloud Firestore emulator v1.16.1, which adds support for read_time in ListCollectionIds.
```

Comment:
Unsure as to why this line is there?

Reply:
ah yep good catch, another stray changelog from a bad merge